swap_meet/item.py

```diff
@@ -1,2 +1,49 @@
+import uuid
+
+# WAVE 2
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0):  # default id is None
```

Comment:
One shouldn't need this comment `# default id is None` to understand that `id` has a default value and it is `None`. A developer would know default param syntax. 

Therefore, we should remove the comment on line 6 because it doesn't explain anything we didn't know. 

Every comment you add to a codebase needs to be maintained. If code logic is updated, then the comment needs to be reviewed and updated if necessary. Therefore you increase the size of the codebase that needs to be maintained, which is more time spent maintaining. 

Comments should be used very sparingly.
```suggestion
    def __init__(self, id=None, condition=0):
```

Reply:
Removed the comment "# default id is None" from line 6.